src/v/pandaproxy/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ v_cc_library(
     logger.cc
     probe.cc
     server.cc
+    kafka_client_cache.cc
```

Comment:
if would be nice if `kafka_client_cache` was added in a separate commit that is ordered before the sharded client cache

Reply:
nit

Reply:
Done.